server/src/pages/shot/text-tool.js

```diff
@@ -52,6 +52,7 @@ exports.TextTool = class TextTool extends React.Component {
   componentDidMount() {
     this.textInput.current.focus();
     previousTextInputWidth = this.textInput.current.clientWidth;
+    previousInputText = "";
```

Comment:
What's the difference between setting the value here vs when the variable was declared?

Reply:
Trying to recall, it was done to reset previousInputText for the next edit